modules/cnv-edit-boot-order-yaml-web.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-boot-order.adoc
+//
+
+[id="cnv-edit-boot-order-yaml-web_{context}"]
+= Editing a boot order list in the YAML configuration file.
+
+Edit the boot order list list in a YAML configuration file by using the CLI.
+
+.Prerequisites
+. You must have access to the OpenShift Container Platform cluster.
```

Comment:
OpenShift Container Platform -> {product-title}

Reply:
Thanks for the input. Will fix and update you when I generate the html again.